spec/acceptance/5_client_spec.rb

```diff
@@ -11,29 +11,33 @@ class { 'keycloak':
       keycloak_realm { 'test': ensure => 'present' }
       keycloak_flow { 'foo on test': ensure => 'present' }
       keycloak_client { 'test.foo.bar':
-        realm                          => 'test',
-        root_url                       => 'https://test.foo.bar',
-        redirect_uris                  => ['https://test.foo.bar/test1'],
-        default_client_scopes          => ['address'],
-        secret                         => 'foobar',
-        login_theme                    => 'keycloak',
-        backchannel_logout_url         => 'https://test.foo.bar/logout',
-        authorization_services_enabled => false,
-        service_accounts_enabled       => true,
-        browser_flow                   => 'foo',
-        roles                          => ['bar_role', 'other_bar_role'],
+        realm                                    => 'test',
+        root_url                                 => 'https://test.foo.bar',
+        redirect_uris                            => ['https://test.foo.bar/test1'],
+        default_client_scopes                    => ['address'],
+        secret                                   => 'foobar',
+        login_theme                              => 'keycloak',
+        backchannel_logout_url                   => 'https://test.foo.bar/logout',
+        backchannel_logout_session_required      => 'true',
+        backchannel_logout_revoke_offline_tokens => 'true',
```

Comment:
For these new parameters, can you verify they are actually getting set?  Further down is where the expected values are confirmed to be set.